templates/_workshopStatusTable.html

```diff
@@ -0,0 +1,43 @@
+{% load wwwtags %}
+
+<table class="table table-striped">
+  <thead>
+    <tr>
+      <th scope="col" style="width: 50%">Warsztaty</th>
+      <th scope="col" style="width: 50%">Status</th>
+    </tr>
+  </thead>
+  <tbody>
+    {% for workshop in your_workshops %}
+    <tr>
+      <th scope="row" style="width: 50%">
+        <a href="{% url 'workshop_edit' workshop.year.pk workshop.name %}">
+          {{ workshop.title }}
+        </a>
+      </th>
+      <td style="width: 50%">
+        {% if workshop.status == 'Z' %}
+          <span class="qualified"> {{ workshop.get_status_display }} </span>
+          {% if workshop.page_content_is_public %}
+            <br><span class="waiting-qualified">Uzupełnij stronę warsztatów!</span>
```

Comment:
This doesn't really make sense on old workshops when you can't feel in forms any more.